### DISCUSSION TITLE
Vercel: An unexpected internal error occurred

### DISCUSSION THREAD

**Discussion by trietly** [vercel/community#1874] [Help]
### Summary

Today I deploy [T3 app](https://create.t3.gg/) to Vercel.
I have tried deploying many times but the internal error message still appears.
these are the packages i use [package.json](https://github.com/trietly/next-edu/blob/main/package.json)

I don't know what the specific error is because of the internal  error message.

Meanwhile, I have tried deploying other apps in Vercel with success. I also tried deploying T3 app locally with success

Thanks for help!

### Additional information

λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
--
13:17:26.917 | ○  (Static)  automatically rendered as static HTML (uses no initial props)
13:17:26.917 | ●  (SSG)     automatically generated as static HTML + JSON (uses getStaticProps)
13:17:26.917 |  
13:17:27.040 | Done in 47.43s.
13:17:28.240 | Traced Next.js server files in: 1.160s
13:17:35.075 | Created all serverless functions in: 6.838s
13:17:35.154 | Collected static files (public/, static/, .next/static): 9.631ms
13:17:36.481 | Build Completed in /vercel/output [2m]

13:17:45.174 | An unexpected internal error occurred
13:17:47.280 | internal_server_error: An unexpected internal error occurred

**Comment by Autushka** [comment]
Having exactly the same issue ((. 

![image](https://user-images.githubusercontent.com/2224124/234280845-6de97bc6-9ce8-443a-896f-b26e104fa37b.png)

**Reply by ndimatteo** [reply]
I started getting this as well... seems to be tied to using `debug={false}` with the `@vercel/analytics` package. The moment I removed that prop from `<Analytics />` I stopped getting these errors at the end of a build.

**Reply by anthill24** [reply]
Having the same issue

**Reply by TechWithTy** [reply]
Same
